### ISSUE TITLE
Uncaught TypeError: Cannot read property 'querySelector' of null

### ISSUE THREAD

**Issue by cwaring** [mark-hahn/markdown-scroll-sync#1]
1. Open markdown file
2. Toggle preview mode, plugin crashes

**Atom Version**: 0.186.0
**System**: Mac OS X 10.10.2
**Thrown From**: [markdown-scroll-sync](https://github.com/mark-hahn/markdown-scroll-sync) package, v0.1.0

### Stack Trace

Uncaught TypeError: Cannot read property 'querySelector' of null

```
At /Applications/Atom.app/Contents/Resources/app/node_modules/q/q.js:128

TypeError: Cannot read property 'querySelector' of null
  at MarkdownScrlSync.startTracking (/Users/cw/.atom/packages/markdown-scroll-sync/lib/main.coffee:39:37)
  at /Users/cw/.atom/packages/markdown-scroll-sync/lib/main.coffee:33:16
  at Emitter.module.exports.Emitter.emit (/Applications/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:82:11)
  at /Applications/Atom.app/Contents/Resources/app/src/pane-container.js:347:34
  at Pane.module.exports.Pane.observeActiveItem (/Applications/Atom.app/Contents/Resources/app/src/pane.js:174:7)
  at /Applications/Atom.app/Contents/Resources/app/src/pane-container.js:346:42
  at Emitter.module.exports.Emitter.emit (/Applications/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:82:11)
  at PaneContainer.module.exports.PaneContainer.setActivePane (/Applications/Atom.app/Contents/Resources/app/src/pane-container.js:218:22)
  at Pane.module.exports.Pane.activate (/Applications/Atom.app/Contents/Resources/app/src/pane.js:638:15)
  at /Applications/Atom.app/Contents/Resources/app/node_modules/markdown-preview/lib/main.js:154:37
  at _fulfilled (/Applications/Atom.app/Contents/Resources/app/node_modules/q/q.js:794:54)
  at self.promiseDispatch.done (/Applications/Atom.app/Contents/Resources/app/node_modules/q/q.js:823:30)
  at Promise.promise.promiseDispatch (/Applications/Atom.app/Contents/Resources/app/node_modules/q/q.js:756:13)
  at /Applications/Atom.app/Contents/Resources/app/node_modules/q/q.js:516:49
  at flush (/Applications/Atom.app/Contents/Resources/app/node_modules/q/q.js:110:17)
  at process._tickCallback (node.js:364:11)

```

### Commands

```
     -3:14.2 core:save (input.hidden-input)
     -2:54.2 core:undo (input.hidden-input)
     -2:45.6 core:save (input.hidden-input)
     -2:10.0 vim-mode:activate-insert-mode (input.hidden-input)
  4x -2:08.7 core:backspace (input.hidden-input)
  2x -2:06.8 core:undo (input.hidden-input)
  2x -1:39.3 core:save (input.hidden-input)
     -1:06.6 vim-mode:activate-insert-mode (input.hidden-input)
     -1:05.9 core:move-right (input.hidden-input)
     -1:02.8 core:backspace (input.hidden-input)
     -0:54.5 core:save (input.hidden-input)
     -0:29.0 settings-view:open (ol.tree-view.full-menu.list-tree.has-collapsable-children.focusable-panel)
     -0:05.1 command-palette:toggle (input.hidden-input)
     -0:03.4 core:select-all (input.hidden-input)
     -0:02.6 core:confirm (input.hidden-input)
     -0:02.6 markdown-preview:toggle (input.hidden-input)
```

### Config

```json
{
  "core": {
    "themes": [
      "one-dark-ui",
      "oceanic-next"
    ],
    "ignoredNames": [
      ".git",
      ".hg",
      ".svn",
      ".DS_Store",
      "Thumbs.db",
      ".meteor"
    ],
    "disabledPackages": [
      "minimap",
      "welcome",
      "git-plus"
    ]
  }
}
```

### Installed Packages

```coffee
# User
autocomplete-plus, v2.4.1
autocomplete-snippets, v1.0.1
docblockr, v0.6.3
emmet, v2.3.4
fancy-new-file, v0.7.0
file-types, v0.3.0
git-control, v0.1.8
language-spacebars, v0.2.2
linter, v0.12.0
linter-csslint, v0.0.11
linter-eslint, v0.4.1
linter-jsonlint, v0.1.2
markdown-format, v2.5.0
markdown-scroll-sync, v0.1.0
meteor-api, v2.12.0
oceanic-next, v0.1.1
react, v0.9.10
terminal-panel, v1.9.2
underscorejs-snippets, v0.2.2
vim-mode, v0.36.0
vim-surround, v0.4.2

# Dev
No dev packages
```

**Comment by mark-hahn** [comment]
Thank you very much for the report.

Fixed in 0.1.1.

**Comment by cwaring** [comment]
Great, thanks for the quick fix!
